### DISCUSSION TITLE
Where is the blog ?

### DISCUSSION THREAD

**Discussion by elabree** [skills/.github#660] [GitHub Pages]
Took the course. In my opinion this website creation is not very easy. Anyhow: after completion of the course I did not see any blog! So, I created some files. But where can I find the pages ???

**Comment by 0ldcamel** [comment]
Have you found your blog? Mine is at my_name.github.io/blog/

**Reply by yingshi-7** [reply]
There's a link at the top of the Settings page.

**Comment by ghost** [comment]
I also didn't find it

**Reply by 0ldcamel** [reply]
Here it is https://yearbook3118.github.io/skills-github-pages/
